### ISSUE TITLE
MA0011/MA0075/MA0076 are not reported for literals

### ISSUE THREAD

**Issue by drieseng** [meziantou/Meziantou.Analyzer#517]
### Version of the Meziantou.Analyzer NuGet package

2.0.45

### Rule Identifier

MA0011

### Target Framework

net7.0

### C# Language version

C# 11

### Description

MA0011/MA0075/MA0076 is not reported for literal values.

### Reproduction Steps

Minimal code:

```c#
Console.WriteLine("#1: " + 1);
Console.WriteLine($"#2: {2}");
Console.WriteLine("#3: " + 3.ToString());
```

### Other information

Personally, I consider this a very low priority issue.

**Comment by drieseng** [comment]
Of course, sorry for wasting your time.
